project/src/main/webapp/create-event.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="create-event.css" />
+    <script src="script.js"></script>
+    <!-- Bootstrap CDN links -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+      />
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+      integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+      crossorigin="anonymous"
+      ></script>
+  </head>
+  <body>
+    <br><br>
+    <h3> Create an Event </h3>
+    <br>
+    <div class="col">
+      <div class="input-group">
+        <div class="input-group-prepend">
+          <span class="input-group-text" id="basic-addon1">Name</span>
+        </div>
+        <input type="text" class="form-control" placeholder="Event Name" aria-describedby="basic-addon1">
+      </div>
+      <br>
+      <div class="row">
+        <div class="col-lg-6">
+          <div class="input-group">
+            <div class="input-group-prepend">
+              <span class="input-group-text" id="basic-addon1">Date</span>
+            </div>
+            <input type="text" class="form-control" placeholder="Event Date" aria-describedby="basic-addon1">
+          </div>
+        </div>
+        <br>
+        <br>
+        <div class="col-lg-6">
+          <div class="input-group">
+            <div class="input-group-prepend">
+              <span class="input-group-text" id="basic-addon1">Time</span>
+            </div>
+            <input type="text" class="form-control" placeholder="Time" aria-describedby="basic-addon1">
+          </div>
+        </div>
+      </div>
+      <br>
+      <div class="inline">
+        <text> Tags: </text>
+        <div class="container">
+          <span class="badge badge-primary text-medium"> Tag 1</span>
+          <span class="badge badge-primary text-medium"> Tag 2</span>
+          <span class="badge badge-secondary text-medium"> + Tag</span>
```

Comment:
Are the leading spaces to provide some kind of spacing? 
If so it might be better for that to go in CSS padding - if not, then it might not be necessary

Reply:
good catch, deleted (done)